DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: syberia
 Type: Package
 Title: Syberia
-Version: 0.6.1
+Version: 0.6.1.1
```

Comment:
Notice that there is a [development version](https://github.com/syberia/syberia/releases). Can you work off that and tag 9001?